sql/smoot_clients_last_seen_1percent_v1.sql

```diff
@@ -0,0 +1,58 @@
+WITH
+  _current AS (
+  SELECT
+    * EXCEPT (submission_date_s3),
+    0 AS days_since_seen,
+    -- For measuring Active MAU, where this is the days since this
+    -- client_id was an Active User as defined by
+    -- https://docs.telemetry.mozilla.org/cookbooks/active_dau.html
+    IF(scalar_parent_browser_engagement_total_uri_count_sum >= 5,
+      0,
+      NULL) AS days_since_visited_5_uri,
+    IF(devtools_toolbox_opened_count_sum > 0,
+      0,
+      NULL) AS days_since_opened_dev_tools,
+    profile_age_in_days AS days_since_created_profile
```

Comment:
We implement a new column here to track days since profile creation based on the last seen 'profile_age_in_days' value.